### PR TITLE
Add Pagination to components page

### PR DESCRIPTION
**Pull request by tyom**
Additionally refactor components page to use consistent breadcrumbs and layout.

Add Example component to contain component examples.

image

### CODE REVIEW

File: src/apps/components/views/_layout.njk [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% extends "_layouts/datahub-base.njk" %}
+
+{% set title = title or getPageTitle()[0] %}
```
Comment (teneightfive) [inline src/apps/components/views/_layout.njk]: What about `heading` or `viewHeading`? Something to distinguish it from the page title that is used in the `<title/>` tag.

File: src/apps/components/views/pagination.njk [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{% extends "_layouts/datahub-base.njk" %}
+
+{% block body_main_content %}
+  <h3 class="heading-medium">Standard</h3>
+  {{
+    Pagination({
+      totalPages: 2,
+      currentPage: 1,
+      prev: null,
+      next: '?term=samsung&page=2',
```
Comment (teneightfive) [inline src/apps/components/views/pagination.njk]: Does the example need `term=samsung`? It's not doing anything to change the rendering of it.

Comment (tyom) [inline src/apps/components/views/pagination.njk]: No it doesn't. I just copied it from tests.

File: assets/stylesheets/components/_.example.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
+@import "../settings";
+
+* + .example {
+  margin-top: $default-spacing-unit * 2;
```
Comment (teneightfive) [inline assets/stylesheets/components/_.example.scss]: Assemblies?

Comment (tyom) [inline assets/stylesheets/components/_.example.scss]: Didn't feel like littering assemblies with dev specific stuff.

**Pull request merged**